src/Nethermind/Nethermind.Evm.Test/Tracing/GethLikeTxTracerTests.cs

```diff
@@ -411,5 +411,42 @@ public void Throw_operation_canceled_after_given_timeout()
 
             Assert.Throws<OperationCanceledException>(() => tracer.StartOperation(0, 0, Instruction.ADD, 0));
         }
+
+        [Test]
+        public void Tracers_cancellation_tokens_does_not_affect_each_other()
+        {
```

Comment:
interesting - in what cases would this test fail? I would like to learn about such scenario